### DISCUSSION TITLE
Viewing Org/Repo Traffic Longer than 14 Days

### DISCUSSION THREAD

**Discussion by ProDCG** [community/community#36740] [Repositories] [answered]
Hi,

Just curious if there was a way to view overall traffic for a repository, as opposed to the last two weeks of information? Specifically within the insights tab I have been looking around in there and haven't found a way to possibly filter for a time length longer then two weeks.

Thanks!

**Comment by radar-blip** [comment, marked as answer]
Just checked. There doesn't seem to be a way to go back any further.

**Reply by ProDCG** [reply]
Darn.
